secv_guis/bimask_app/main_window.py

```diff
@@ -774,6 +776,10 @@ def _handle_img_selection(self, basename):
         abspath = os.path.join(self.file_lists.img_list.dirpath, basename)
         success = self.graphics_view.new_image(abspath, self.mask_color,
                                                self.preannot_color)
+        if self.file_lists.preannot_list is not None:
+            print(basename)
+            self.file_lists.preannot_list.update_path(self.file_lists.preannot_list.dirpath,basename)
+
```

Comment:
I don't quite understand what is happening here, could you explain?

Reply:
It's about filtering the the other lists by the name of the selected image